.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+## What happened?
+
+Describe the problem.
```

Comment:
Maybe we should be more explicit here about the fact they should give us details here instead of very vague descriptions like "doesn't work" or "can't login"?